src/app/main/main.controller.js

```diff
@@ -39,14 +39,22 @@ angular.module('transmartBaseUi')
      * Activate tab
      * @param tabTitle
      * @param tabAction
-       */
+     */
     $scope.activateTab = function (tabTitle, tabAction) {
-        $scope.tabs.forEach(function (tab) {
-            tab.active = tab.title === tabTitle;
-        });
-        $state.go('workspace', {action:tabAction});
+      $scope.tabs.forEach(function (tab) {
+        tab.active = tab.title === tabTitle;
+      });
+      $state.go('workspace', {action:tabAction});
```

Comment:
Stateparams are already used here, better to stick with consistency and implement it fully